app.py

```diff
@@ -110,6 +118,24 @@ def main():
             EPICGAMES_APP_IDS, epicgames_notifier, IGNORE_FIRST_NOTIFICATION
         )
 
+    if WATCH_GOG:
+        gog_notifier = Discord(
+            webhook_url=DISCORD_WEBHOOK_URL,
+            role_ids=DISCORD_MENTION_ROLE_IDS,
+            user_ids=DISCORD_MENTION_USER_IDS,
+            platform="GOG",
+            # TODO update this if necessary
+            thumb_url=(
+                #"https://github.com/kurokobo/game-update-notifier/raw/main/"

```

Comment:
You can leave this as this is, but it would be helpful if you could change it to the URL of my repository.

Reply:
I'll leave this for you to change after merging, I've fixed everything else